pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -573,9 +573,14 @@ in
   };
 
   pg = attrs: {
-    buildFlags = [
-      "--with-pg-config=${postgresql}/bin/pg_config"
-    ];
+    # Force pkg-config lookup for libpq.
+    # See https://github.com/ged/ruby-pg/blob/6629dec6656f7ca27619e4675b45225d9e422112/ext/extconf.rb#L34-L55
+    #
+    # Note that setting --with-pg-config=${postgresql}/bin/pg_config would add
+    # an unnecessary reference to the entire postgresql package.
+    buildFlags = [ "--with-pg-config=ignore" ];
+    nativeBuildInputs = [ pkg-config ];
```

Comment:
just for reference, https://github.com/NixOS/nixpkgs/pull/234470 once that gets merged we can just switch to use libpq instead of postgresql

Reply:
Thanks, subscribed to PR updates.